### DISCUSSION TITLE
Deno compile executable details manifest file

### DISCUSSION THREAD

**Discussion by gewoonwoutje** [denoland/deno#9509] [Ideas]
I'm a big fan of the deno compile command. It's never been easier to create a single-executable of my code, and it's amazingly fast.

Though I noticed the output shows up with a deno logo, 
![image](https://user-images.githubusercontent.com/9407072/107891420-d77e1b00-6f1e-11eb-9bb1-19a487cd73d6.png)
and the properties show it as;
![image](https://user-images.githubusercontent.com/9407072/107891426-dc42cf00-6f1e-11eb-9a39-17fe37bceb2f.png)

Could this be a potential security risk (one could distribute a malicious copy of deno)? I also wonder if this could have legal consequences.

Also, as a user, I'd like to be able to sign it with my own (company's) name. Is changing these properties already possible? I think it might make sense to give a manifest file as parameter with application details to override these values.

**Comment by kitsonk** [comment]
Duplicate of #9395
